op-monitorism/faultproof_withdrawals/cli.go

```diff
@@ -62,14 +65,20 @@ func CLIFlags(envVar string) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:    L2NodeURLFlagName,
-			Usage:   "L2 rollup node consensus layer (op-node) URL",
+			Usage:   "[DEPRECATED] L2 rollup node consensus layer (op-node) URL - this flag is ignored",
 			EnvVars: opservice.PrefixEnvVar(envVar, "L2_OP_NODE_URL"),
+			Hidden:  true,
```

Comment:
I assume this is desired to add 'true' here? 
maybe we should for the first version keep the flag as deprecated for a certain period of time before putting this one as hidden immediately for people to avoid to be lost in case of an issue.